tools/build_versions.py

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env python
 
-import numpy as np
-import scipy as sp
-import matplotlib as mpl
-from PIL import Image
-import Cython
-import networkx
+from pathlib import Path
+import requirements
+import importlib
```

Comment:
Few remarks more:
- `import importlib` doesn't look neccessary;
- Where `__file__` comes from?

Reply:
`__file__` is builtin isn't it?

Reply:
I think with the `__name__ == '__main__'` the script is more correct now.